src/pages/suminKim/Login/Login.js

```diff
@@ -1,9 +1,84 @@
 import React from 'react';
+import './Login.scss';
+import '../../../styles/common.scss';
+import '../../../styles/reset.scss';
+import { Link } from 'react-router-dom';
+
+//     // <!-- 글꼴 -->
+//     <link rel="preconnect" href="https://fonts.googleapis.com">
+//     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+// @import<link href="https://fonts.googleapis.com/css2?family=Lobster&display=swap" rel="stylesheet">
 
 class Login extends React.Component {
+  goToMain = () => {
+    this.props.history.push('/Main');
+  };
+
+  constructor() {
+    super();
+    this.state = {
+      idValue: '',
+      pwValue: '',
+      buttonOn: false,
+    };
+  }
+  handleIdInput = e => {
+    this.setState({
+      idValue: e.target.value,
+    });
+  };
+  handlePwInput = e => {
+    this.setState({
+      pwValue: e.target.value,
+    });
+  };
+  buttonChange = () => {
+    const { idValue } = this.state;
+    const { pwValue } = this.state;
```

Comment:
```suggestion
    const { idValue, pwValue } = this.state;
```
한번에 구조분해할당 할 수 있습니다.